### PR TITLE
fix(modal): adjust zIndex for popovers and tooltips

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Adjusted Console zIndex tokens so tooltips and popovers appear correctly
- [x] Updated snapshots
- [x] Fixed fill and stroke bug in tooltip and popover arrow

### CODE REVIEW

File: packages/paste-libraries/reakit/src/Popover/__keys.ts [hunk 1]
```diff
@@ -26,6 +26,6 @@ const POPOVER_STATE_KEYS = [
   'place',
 ] as const;
 export const POPOVER_KEYS = POPOVER_STATE_KEYS;
-export const POPOVER_ARROW_KEYS = [...POPOVER_KEYS, 'size'] as const;
+export const POPOVER_ARROW_KEYS = [...POPOVER_KEYS, 'size', 'fill', 'stroke'] as const;
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-libraries/reakit/src/Popover/__keys.ts:29]: This fixes a regression where the popover and tooltip beak weren't being passed the fill or stroke props.

[the remark this answers is not in the document]
Comment (SiTaggart) [reply packages/paste-libraries/reakit/src/Popover/__keys.ts:29]: To save modifying this generated file from Reakit can we spread in the additional props in our `PopoverArrow.tsx` file?

```
import {POPOVER_ARROW_KEYS} from './__keys';

const POPOVERARROWKEYS = [...POPOVER_ARROW_KEYS, 'fill', 'stroke'];

```

[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-libraries/reakit/src/Popover/__keys.ts]: That change has been made.

File: packages/paste-core/components/modal/stories/index.stories.tsx [hunk 2]
```diff
@@ -453,4 +455,79 @@ storiesOf('Components|Modal', module)
...
+        </Button>
+        <Modal ariaLabelledby={modalHeadingID} isOpen={isOpen} onDismiss={handleClose} size="default">
+          <ModalHeader>
+            <ModalHeading as="h3" id={modalHeadingID}>
+              Modal Heading
+            </ModalHeading>
+          </ModalHeader>
+          <ModalBody>
+            <Paragraph>Look at the modal work with a tooltip.</Paragraph>
+            <Tooltip text="Welcome to Paste!">
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-core/components/modal/stories/index.stories.tsx]: I've set a delay on this story and the popover one so they show visible when the page loads.

[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-core/components/modal/stories/index.stories.tsx]: No more delay. Got them working with `waitBeforeScreenshot` from eyes.

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**